Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM ubuntu:xenial
 
 LABEL maintainer='Chris Rust <chris.rust@solarwinds.com>'
 
-ENV APPOPTICS_SNAPTEL_VERSION '2.0.0-ao1.1919'
+ARG APPOPTICS_SNAPTEL_VERSION='2.0.0-ao1.1919'
```

Comment:
Can args be read as environment variables as well?
If not should be something like
```
ARG APPOPTICS_SNAPTEL_VERSION='2.0.0-ao1.1919'
ENV APPOPTICS_SNAPTEL_VERSION=${APPOPTICS_SNAPTEL_VERSION}
```

Reply:
Hey Beau how is it going?
During build the ARG is available as if it is an environment variable.
Outside of it, is that environment variable elsewhere during runtime? I [can't](https://github.com/appoptics/appoptics-agent-docker/search?q=APPOPTICS_SNAPTEL_VERSION&unscoped_q=APPOPTICS_SNAPTEL_VERSION) find it. 

Is it required that the version is available as environment variable?

Reply:
Aye I just checked the repo, looks like its only used for the package manager in the dockerfile 👍